### ISSUE TITLE
PaddleSeg训练报错

### ISSUE THREAD

**Issue by xingyun8** [PaddlePaddle/Paddle#50803]
### 请提出你的问题 Please ask your question

使用服务器上的这个预训练模型pretrained: https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz，训练显示The pretrained model directory is not Found: /home/qq/.paddleseg/pretrained_model/deeplab_mobilenet_fp32_fluid/model.pdparams

**Comment by xingyun8** [comment]
以下是我训练使用的完整的yml文件

batch_size: 4
iters: 1000

train_dataset:
  type: OpticDiscSeg
  dataset_root: data/cc
  transforms:
    - type: Resize
      target_size: [513, 513]
    - type: RandomHorizontalFlip
    - type: Normalize
  mode: train

val_dataset:
  type: OpticDiscSeg
  dataset_root: data/cc
  transforms:
    - type: Resize
      target_size: [513, 513]
    - type: Normalize
  mode: val

optimizer:
  type: sgd
  momentum: 0.9
  weight_decay: 4.0e-5

lr_scheduler:
  type: PolynomialDecay
  learning_rate: 0.01
  end_lr: 0
  power: 0.9

loss:
  types:
    - type: CrossEntropyLoss
  coef: [1]

# distill_loss is used for distillation
distill_loss:
  types:
    - type: KLLoss
  coef: [3]

model:
  type: DeepLabV3P
  backbone:
    type: MobileNetV2_x1_0 
    pretrained: https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz

**Comment by xingyun8** [comment]
@Shixiaowei02 您好，能抽空帮忙看下这个问题吗，多谢。

**Comment by xingyun8** [comment]
@Shixiaowei02 大佬，您好，能抽空帮忙看一下这个问题吗？

**Comment by Shixiaowei02** [comment]
您好，看起来是文件路径没有设置正确，按照路径字符串找不到所需的文件

**Comment by xingyun8** [comment]
> 您好，看起来是文件路径没有设置正确，按照路径字符串找不到所需的文件

您好，我把这个https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz文件下载下来之后，里面没有model.pdparams这个文件，请问是怎么回事呀？

**Comment by xingyun8** [comment]
@Shixiaowei02  您好，我把这个[https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz
文件下载下来之后，里面没有model.pdparams这个文件，请问是怎么回事呀？

**Comment by xingyun8** [comment]
@Shixiaowei02 您好，我把这个https://paddlelite-demo.bj.bcebos.com/models/deeplab_mobilenet_fp32_fluid.tar.gz
文件下载下来之后，里面没有model.pdparams这个文件，请问是怎么回事呀？
